polytope/polytope.py

```diff
@@ -201,19 +201,40 @@ def __copy__(self):
         P.fulldim = self.fulldim
         return P
 
+    def contains(self, point, abs_tol=ABS_TOL):
+        """Return True if polytope contains points.
+
+        @type point: array_like, 1d array, or 2d array (of column vectors)
+
+        @rtype: bool, 1d array
+
+        See Also
+        ========
+        L{is_inside}
+        """
+        if not isinstance(point, np.ndarray):
+            point = np.array(point)
+        if point.ndim == 1:
+            point.shape = (point.size, 1)
+        assert point.shape[0] == self.dim, "points should be col vectors"
+        test = self.A.dot(point) - self.b.reshape((self.b.size, 1)) < abs_tol
+        return np.all(test, axis=0)
+
     def __contains__(self, point, abs_tol=ABS_TOL):
         """Return True if polytope contains point.
 
         See Also
         ========
-        L{is_inside}
+        L{is_inside}, L{Polytope.contains}
         """
         if not isinstance(point, np.ndarray):
             point = np.array(point)
         test = self.A.dot(point.flatten()) - self.b < abs_tol
         return np.all(test)
 
     def are_inside(self, points, abs_tol=ABS_TOL):
+        warnings.warn('polytope.are_inside is deprecated.'
```

Comment:
I added the method `are_inside`, [in 2014](https://github.com/tulip-control/tulip-control/commit/395998b5e6e33b5fdd56cfcdd9312bfffc2c9d19). Regarding the name, I am undecided between `contains` and `are_inside`. The first one is shorter, simpler, doesn't raise the singular vs plural issue, doesn't require pressing underscore, so it seems to be a better choice.

One concern may be the container-related connotation that its name suggests. Also, "inside" explicitly indicates that this is a strict inequality check (`<`), excluding the boundary. The plural signifies the main difference between `__contains__` and `are_inside` (which not as evident when comparing `__contains__` to `contains`, i.e., considering whether the method name is self-explanatory or not when reading code).

Of course, `contains(point)` is awkward. (This choice reminds of the "exists" vs "exist"---"forall" doesn't raise this issue).